src/Amadevus.RecordGenerator.Generators/EqualityGenerators.cs

```diff
@@ -170,40 +170,50 @@ private static MemberDeclarationSyntax GenerateGetHashCode(RecordDescriptor desc
                 .AddModifiers(
                     PublicKeyword,
                     OverrideKeyword)
-                .AddBodyStatements(
-                    CheckedStatement(UncheckedStatement)
-                    .AddBlockStatements(
-                        LocalDeclarationStatement(
-                            VariableDeclaration(
-                                IdentifierName(varKeyWord))
-                            .AddVariables(
-                                VariableDeclarator(
-                                    Identifier(hashCodeVariableName))
-                                .WithInitializer(
-                                    EqualsValueClause(
-                                        LiteralExpression(
-                                            NumericLiteralExpression,
-                                            Literal(hashCodeInitialValue)))))))
-                    .AddBlockStatements(
-                        descriptor.Entries.Select(EntryHashCodeRecalculation)
-                        .ToArray())
-                    .AddBlockStatements(
-                        ReturnStatement(
-                            IdentifierName(hashCodeVariableName))));
+                .AddBodyStatements(expression);
+            
+            StatementSyntax SinglePropertyGetHashCode(RecordDescriptor rd)
```

Comment:
The parameter is unused, and also unnecessary.

Reply:
It isn't recommended to use captured variables if not necessary, as you have done further up the class.

Reply:
> It isn't recommended [...]

Where?

Reply:
Closures cause the compiler to generate a struct to capture the variable.  In this case it makes the code no clearer and is needless.  I am also explicitly following your own code style in the very same file.

Reply:
You prefer the code easier to read when discussing line breaks, but here you opt to decrease readability by using an obscure `rd` variable name, renaming the already available parent method's parameter for the sake of avoiding display class?

I'm not concerned with performance here. It's a code generator. It creates a completely separate, additional compilation.

Reply:
> By not using closures, I am explicitly following your own code style in the very same file.

There are no other places in this file where the local function parameter is available in parent method (that I see).